helix-term/src/ui/picker.rs

```diff
@@ -556,6 +556,22 @@ impl<T: Item + 'static> Component for Picker<T> {
             ctrl!('t') => {
                 self.toggle_preview();
             }
+            ctrl!('y') => {
+                if let Some(contents) = self.selection() {
+                    let contents = contents.sort_text(&self.editor_data).to_string();
```

Comment:
(Oops, I forgot to add the comment. Here it is.)

Since the sort text is not always equal to the stringified label, maybe we specifically want `String::from(contents.label(&self.editor_data))` so that it always copies what is actually rendered in the picker minus the style?